index.js

```diff
@@ -26,42 +28,63 @@ const test = (whatWeTest, actualResult, expectedResult) => {
 
 // Functions
 
-const getType = (value) => {
-    // Return string with a native JS type of value
-};
+const getType = (value) => typeof value;
 
-const getTypesOfItems = (arr) => {
-    // Return array with types of items of given array
-};
+const getTypesOfItems = (arr) => arr.map((element) => getType(element));
```

Comment:
Можно даже так для краткости:
```suggestion
const getTypesOfItems = (arr) => arr.map(getType);
```
Но у тебя безопаснее, т.к. в `getType` не приедут доп. аргументы, которые передаёт `map`.

Reply:
Не знал, что можно так коротко записывать. Так как безопаснее, оставил свой вариант.